### ISSUE TITLE
text in history not visible with dark theme

### ISSUE THREAD

**Issue by vnijs** [jupyterlab/jupyterlab-git#410]
## Description

Text in history not visible with dark theme

![image](https://user-images.githubusercontent.com/2998956/64562837-ac8fa080-d302-11e9-9b79-0d83191ae0ea.png)

## Reproduce

Select dark theme in jupyterlab and click on History in a git repo and then on a commit

## Expected behavior

Visible text related to file history

## Context

```{r}
jovyan@mini ~ pip3 show jupyterlab-git
Name: jupyterlab-git
Version: 0.8.1
Summary: A server extension for JupyterLab's git extension
Home-page: UNKNOWN
Author: Git Intern Team - Noah Stapp, Jenna Landy, Alena Mueller; AWS - Jaipreet Singh, Neelam Gehlot; Quansight - Saul Shanabrook
Author-email: None
License: UNKNOWN
Location: /usr/local/lib/python3.6/dist-packages
Requires: notebook, nbdime
jovyan@mini ~ jupyter labextension list
JupyterLab v1.1.2
Known labextensions:
   app dir: /usr/local/share/jupyter/lab
        @jupyter-widgets/jupyterlab-manager v1.0.2  enabled  OK
        @jupyterlab/git v0.8.2  enabled  OK
        @ryantam626/jupyterlab_code_formatter v0.5.1  enabled  OK
        jupyterlab-jupytext v1.0.2  enabled  OK
        jupyterlab-server-proxy v2.0.0  enabled  OK*
        nbdime-jupyterlab v1.0.0  enabled  OK

   local extensions:
        jupyterlab-server-proxy: /tmp/jupyter-server-proxy/jupyterlab-server-proxy
jovyan@mini ~ 
```

There was were some message in the JS console when I clicked on the git button on the top-right of the file browser but I'm not sure that is related to the font color issue.

![image](https://user-images.githubusercontent.com/2998956/64572673-33517700-d31d-11e9-9d54-75d7982a9f9b.png)

**Comment by fcollonval** [comment]
@vnijs could you open a separate issue for the second bug please?
